packages/config/config/devices/0x031e/vzw32-sn.json

```diff
@@ -706,40 +723,44 @@
 			"unsigned": true
 		},
 		{
-			"#": "109[0xff000000]",
-			"label": "After Hours Time Range Start Hour",
-			"description": "The UTC hour where After Hours mode should start",
+			"#": "109[0xff]",
+			"label": "Future Use",
```

Comment:
Remove please. We have the option to get/set arbitrary config parameters - no need to document ones that should not be used.

Reply:
Ohh, I see. That's fine for now then. We'll need to add a way to effectively hide config parameters from users.